src/app/fyle/add-edit-expense/add-edit-expense.page.ts

```
@@ -3518,14 +3518,13 @@ export class AddEditExpensePage implements OnInit {
         const isAdvanceWalletEnabled = res.orgSettings?.advances?.advance_wallets_enabled;
         let customProperties = res.customProperties;
         customProperties = customProperties.map((customProperty) => {
+          if (!customProperty.value) {
+            customProperty = this.customFieldsService.setDefaultValue(customProperty, customProperty.type);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)